sfcasts/list-orders.md

```diff
@@ -1,117 +1,64 @@
-We can paginate it by adding `’page' => ['size' => 5]` to the query.`
-Now refresh - nothing changed, but that's because I only have 5 orders before.
-Let me quickly buy one more product behind the scene and if I update again 
-It's still five, but here's the new one in the beginning.
-So our pagination works and only last 5 are shown!
-
-Ideally, we should add a real pagination below to allow users to navigate over their orders without leaving our website, but for now let’s just go the extra mile and add a link to the full list of orders in LS.
-
-In the template, add: `<a href="https://app.lemonsqueezy.com/my-orders" target="_blank">More Orders</a>`
+Lemon Squeezy returns *10* orders by default. If you want to see fewer than ten at a time, we can paginate the list by adding `'page' => ['size' => 5]` to the query. If we head back and refresh... hm... nothing changed. Ah! That's because I only had *five* orders before. I'll buy another lemonade behind the scenes... refresh the page again, and... we *still* only see five, but that's because the sixth order - the oldest - is being paginated. It's working!
 
-But wait, should we always show that link? Nope. Only if the user has more orders than we rendered. Let’s dump the $orders again and inspect the dump, specifically the `meta` key. We get `total` and `perPage`.
+*Ideally*, we should add *real* pagination below so users can navigate through all of their orders without leaving our site, but for now, let’s just add a link to the full list of orders in LemonSqueezy.
 
-So wrap the link with: `{% if orders.meta.page.total > orders.meta.page.perPage %}`
+In the template, add:
 
-Nice! But… click the link… and wait… the page is empty? Yeah, it does not work in test mode, and I hope LemonSqueezy will fix it soon.
-That’s because LemonSqueezy only shows production orders here but we’re in test mode. Oof. Thought there’s a workaround.
+`<a href="https://app.lemonsqueezy.com/my-orders" target="_blank">More Orders</a>`.
 
-## Refer to All Orders in LS (Even in Test Mode)
+But we don't need to see this link *all the time* - only if the user has more than five orders. Let's `dd($orders)` again, go refresh, and inspect the data. In `meta` `page`, we see `total` and `perPage`, so head back, remove the `dd()`, and wrap the link with `{% if orders.meta.page.total > orders.meta.page.perPage %}`. I'll fix this spacing and add `{% endif %}` at the end.
 
-Let’s pass the latest order to the template: `’latestOrder' => $orders['data'][0] ?? null,`. Now update the link:
-`<a href="https://app.lemonsqueezy.com/my-orders/{{ (orders.data|first).attributes.identifier|default('') }}" target="_blank">More Orders</a>`
+Okay, if we refresh again and click the link... wait... the page is *empty*? Yeah... *about that* - this doesn't work in test mode because LemonSqueezy only shows *production* orders here. I'm hopeful LemonSqueezy will fix that soon, but for now, we can only see this in action on production.
```

Comment:
```suggestion
Okay, if we refresh again and click the link... only the latest order on the list? Yeah... *about that* - this doesn't work in test mode because LemonSqueezy only shows *production* orders here. I'm hopeful LemonSqueezy will fix that soon, but for now, we can only see the orders we link directly on this page or see this in action in production mode.
```